src/components/pagination-arrow/pagination-arrow.tsx

```diff
@@ -6,30 +7,49 @@ const ARROW_LEFT_PATH =
 const ARROW_RIGHT_PATH =
   'M12.2929 9.29289C11.9324 9.65338 11.9047 10.2206 12.2097 10.6129L12.2929 10.7071L17.585 16L12.2929 21.2929C11.9324 21.6534 11.9047 22.2206 12.2097 22.6129L12.2929 22.7071C12.6534 23.0676 13.2206 23.0953 13.6129 22.7903L13.7071 22.7071L19.7071 16.7071C20.0676 16.3466 20.0953 15.7794 19.7903 15.3871L19.7071 15.2929L13.7071 9.29289C13.3166 8.90237 12.6834 8.90237 12.2929 9.29289Z';
 
-const PaginationArrow = ({onClick, disabled, id, path}: {onClick: () => void; disabled: boolean; id: string; path: string}) => {
+enum ARROW_TYPE {
+  LEFT = 'left',
+  RIGHT = 'right'
+}
+
+interface PaginationArrowProps {
+  onClick: () => void;
+  onEnter: () => void;
```

Comment:
where do you use onEnter? 
doesn't onClick include it?

Reply:
you're right, i'll remove it